src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -238,13 +238,19 @@ Blank()
 >> Text(TimeValue("15:06"))
 "3:06 PM"
 
+// Result will depend on .Net version
 >> Text(Time(17,2,3), DateTimeFormat.LongTime, "vi-VN")
+.Net5: "17:02:03"
```

Comment:
The answer here should depend on the language semantics, not the .net version. (ie, it should be the same across .net and JS). 
This may mean we need to add a check on Text() function, etc. Similar to what we had ot do for Win 11 on French, #1577